src/cmd/CMakeLists.txt

```diff
@@ -29,3 +29,10 @@ add_custom_command(TARGET zilliqa
     COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:genkeypair> ${CMAKE_BINARY_DIR}/tests/Zilliqa)
 target_include_directories(genkeypair PUBLIC ${CMAKE_SOURCE_DIR}/src)
 target_link_libraries(genkeypair PUBLIC Crypto)
+
+add_executable(sign sign.cpp)
```

Comment:
As per https://github.com/Zilliqa/Issues/issues/177, perhaps we could have a discussion of the naming (not only this) @jiayaoqijia @bb111189 and others who have any thoughts.